### ISSUE TITLE
Android: App turns solid color after two seconds

### ISSUE THREAD

**Issue by runkelruebe** [standardnotes/mobile#510]
Since the update to version 3.9.1 the app turns grey (just a solid color) and shows nothing, no top bar, no list of notes, etc after about 2 seconds.

Steps to recreate:

1. start the app
2. the app loads fine, it shows the top bar, burger menu and the list of notes, it says "decrypting notes" and "syncing..."
3. after one or two seconds when the syncing is done everything turns grey, button presses do nothing
 
I cleared cache and data, reinstalled the app, restarted my phone in all possible combinations but the same thing happens every time.

I would take a screenshot but it's disallowed by the app (afaik I never changed that setting).

OnePlus 7 Pro
Android 11
Oxygen OS 11.0.3.1.GM21BA

**Comment by TomWaitsForNoMan1** [comment]
I have the same problem on my Galaxy S20+. Only started happening this morning

**Comment by kaktusztea** [comment]
Same problem on iPhone SE 2020.
Just started at this moment, I used it frequently in this day before and before that and no problem that time.

**Comment by jbeldia** [comment]
Hi @runkelruebe, I'm very sorry for the inconvenience. We'll need to investigate further, so in case this is an account specific issue, please let me know your account email by sending us an email at help@standardnotes.com.

@kaktusztea please send us an email as well!

**Comment by runkelruebe** [comment]
@jbeldia Thank you, I just sent you an email.

**Comment by kaedech** [comment]
I've been experiencing the same problem for the past few days.
The screen will turn to a single color after the "Syncing..." is complete.
The screen color will be white for the light theme and black for the dark theme.

OS version: iOS 15.1
App version: 3.9.1

**Comment by moughxyz** [comment]
Can you try version 3.9.3? You'll have to join our Testflight to get it now while we wait for Apple to approve: https://testflight.apple.com/join/H61MmmCA

**Comment by zzzpppzzz** [comment]
I am having the exact same problem as well. However, I am on iOS. Happens on both of my iOS device. One on iOS 14 and one on iOS 13.

Also, the latest versions, 3.9.2 and 3.9.3 do not fix the issue. The exact same issue persists.

(Also, I am unable to enable error reporting as I can not get into the menu before the screen goes blank)

**Comment by kaedech** [comment]
@moughxyz 
In my environment, versions 3.9.2 and 3.9.3 fixed this issue.

OS version: iOS 15.1
App version: 3.9.2 and 3.9.3 (App Store version)

**Comment by runkelruebe** [comment]
@jbeldia After an update of the "Android System WebView" (current version: 96.0.4664.45) Standard Notes seems to be working again as the screen doesn't turn into a blank area anymore and it syncs just fine.

**Comment by GlennSchner** [comment]
> @jbeldia After an update of the "Android System WebView" (current version: 96.0.4664.45) Standard Notes seems to be working again as the screen doesn't turn into a blank area anymore and it syncs just fine.

Same issue on my Samsung Galaxy Note 8, issue still persists on ASW v96.0.4664.45. Has anyone checked if downgrading Standard Notes fixes the issue?

**Comment by RavetcoFX** [comment]
3.9.3 was also broken for me on my S20 FE 5G SM-G781W running One UI 3.1 Oct 1 2021 Patch
Downgrading to 3.9.2 resolves the issue for now, but I had to disable auto updating in Google Play.
This is a critical breaking bug that meant I had to jot things down quickly in Samsung Notes.

**Comment by dreamflasher** [comment]
I can confirm this bug on 3.9.2, F-Droid

**Comment by moughxyz** [comment]
Could anyone experiencing this issue try these steps:

1. Create a backup of your existing account using the web app
2 Register for a new account and import that backup
3. On mobile, sign out of your old account (perhaps by uninstalling the app) then sign into the new account.

Does it still go blank in this case? If it does this will let me know that it’s a data related issue rather than device related issue.

**Comment by javathehunt** [comment]
So this will go unfixed. Can anyone recommend a replacement to standard notes?

**Comment by moughxyz** [comment]
Did no one want to help debug? https://github.com/standardnotes/mobile/issues/510#issuecomment-979279849

**Comment by javathehunt** [comment]
> Did no one want to help debug? [#510 (comment)](https://github.com/standardnotes/mobile/issues/510#issuecomment-979279849)

So I tried this. I deleted the app from the android. I reinstalled it. When opening the app it takes me to a white screen on top says Authentication Requires / Keychain Wiped - We detected your keychain was wiped this happens when restoring from a backup please enter account password to restore account keys. Underneath it has a entry box for password and it says "waiting." It is not allowing me to do anything else other than enter password for the old account. I cannot get out of that screen to log in with new account that I already created on the desktop. Please advise how to scrub old data / keys the app will not allow me to proceed.

**Comment by dreamflasher** [comment]
Go in Android Settings → Apps → Standardnotes and "delete all data" and "delete cache".

**Comment by javathehunt** [comment]
All seems to be working currently. 

Cleared both app Data and Cache.
Allowed me to log in with new credentials that I created on the desktop.
The app opened I created a few notes.
Imported the backup on the desktop and it populated the handheld.
Opened and closed the app a few times edited several notes.
Rebooted the device. Edited more notes.
All seems stable at this time.
I don't know what this means since it was the update that broke the app and I imported the same data that was in use before so not sure how this helps explain.

Running V 3.9.3. I do not know if this is the same broken version that was in there previously. Like others experienced as soon as that update came thru I opened the app and the screen went grey in two seconds would not allow me past that point. I saw the thread here so I have been waiting for a fix to push thru but since that didn't happen I posted today to see if a fix was in the works.  Thank you all.

Moto E5
Oreo 8.1.0

**Comment by moughxyz** [comment]
@javathehunt so this is with the new temp account right? What about signing into your previous account that originally caused the issue?

**Comment by javathehunt** [comment]
- That was with the new account.
- So I just opened the app and I was logged into the new account. 
- I logged out of that account went to the login screen entered the credentials for the old account and logged in.
- Two seconds later I get the grey screen just like before so I close app but stay logged in. 
- I then go to the  computer browser log into the old account and edit one note.
- Go  back to the android app open it again and this time it does not give me the grey screen but allows me into the app. I see the modified note I just edited on the desktop.
- I log out of the account log back into the new account then log out of that one and back into the old account again and all seems ok. No more grey screen of death. All because of what? Because I logged into the desktop and modified a note and that fixed the grey screen? I doubt it because during the two week period when I could not use the phone app I still made changes to the desktop notes and periodically tried to access the phone app and was still getting the grey screen. Though I never tried to log out of the phone and back in during that two week period. Some reason, it is all working now. But it makes me want to go back to my 7-ring Daytimer.
